### PR TITLE
#146683081 Front end unit testing setup

### PR DESCRIPTION
**Pull request by Hector101**
#### What does this PR do?
* This PR sets up a front end testing frameworks to the project using Jasmine and Karma

#### Description of Task to be completed?
* Task to be completed installs the required dependencies including - jasmine, jasmine-core, 
karma, karma-chrome-launcher, karmer-cli, and karma-jasmine.
* initialize karma to the project using the command "./node_modules/.bin/karma init" to choose and setup the preferred testing framework, in this case Jasmine and setup the platform to run test, in this case Chrome Browser.

#### How should this be manually tested?
* Run the command  "./node_modules/.bin/karma start" to run test on Chrome browser
* Run test with jasmine using the command "./node_modules/.bin/jasmine"

#### Any background context you want to provide?
* N/A
#### What are the relevant pivotal tracker stories?
* 146683081 Setup unit testing for the front end
#### Screenshots (if appropriate)
* Testing mock file with jasmine
screen shot 2017-06-12 at 1 06 29 pm

* Mock test file 
screen shot 2017-06-12 at 1 06 47 pm

* Chrome browser testing that karma is working properly
screen shot 2017-06-12 at 1 16 09 pm

#### Questions:
*N/A

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,7 @@
+MONGOHQ_URL=localhost
+FB_CLIENT_ID=
+FB_CLIENT_SECRET=
+TWITTER_CONSUMER_KEY=
+TWITTER_CONSUMER_SECRET=
+GITHUB_CLIENT_ID=
+GITHUB_CLIENT_SECRET=
```
Comment (tonerdo) [inline .env]: Env keys without values are not ideal. It's either they have a dummy value or they aren't there at all

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "plugins": [
+      "import"
+    ],
+    "env": {
+      "node": true,
+      "es6": true,
+      "browser": true,
+      "mocha": true,
+    },
+      "rules": { "one-var": 0,
```
Comment (tonerdo) [inline .eslintrc.js]: Poor formatting. `"one-var": 0` should be on a new line

**Review by tonerdo — CHANGES_REQUESTED** [review]
A few comments but I think we're good. You should probably listen to hound-ci as well

**Review by tonerdo — CHANGES_REQUESTED** [review]
A few comments but I think we're good. You should probably listen to hound-ci as well

**Comment by temilaj** [comment]
Testing bot integration. Ignore this PR

**Pull request closed without merging**